Core/GameEngine/Source/Common/System/XferCRC.cpp

```diff
@@ -140,30 +126,35 @@ void XferCRC::xferSnapshot( Snapshot *snapshot )
 void XferCRC::xferImplementation( void *data, Int dataSize )
 {
 
-	if (!data || dataSize < 1)
-	{
-		return;
-	}
-
 	const UnsignedInt *uintPtr = (const UnsignedInt *) (data);
+	int validData = (data != NULL) & (dataSize > 0);
```

Comment:
Perhaps just set dataSize to 0 if data is NULL, then `validData` can probably be removed.